File: 2st/Riyeon/숨어있는 숫자의 덧셈 (2).py
def solution(my_string):
    for i in my_string:
        if not i.isdigit():
            my_string = my_string.replace(i, ' ')
    li = my_string.split(' ')
    answer = 0
    for i in li:
        if len(i) != 0:
            answer += int(i)
    return answer

File: 3st/Juhee/숨어있는 숫자의 덧셈 (2).java
class Solution {
    public int solution(String my_string) {
        int answer = 0;
        String[] str = my_string.replaceAll("[a-zA-Z]", " ").split(" ");
        
        for (String s : str) {
            if (!s.equals("")) {
                answer += Integer.parseInt(s);
            }
        }
        
        return answer;
    }
}
